packages/websockets/interfaces/gateway-metadata.interface.ts

```diff
@@ -36,7 +36,7 @@ export interface GatewayMetadata {
   connectTimeout?: number;
   /**
    * How many ms without a pong packet to consider the connection closed
-   * @default 5000
+   * @default 20000
```

Comment:
[indeed](https://github.com/socketio/engine.io#methods-1)

![image](https://user-images.githubusercontent.com/13461315/155040623-6317b24a-9f7b-4361-895d-162a96c71eaf.png)

---

What about using the [numeric separator](https://www.proposals.es/proposals/Numeric%20separators) here _(and in all other places on this file)_

```suggestion
   * @default 20_000
```

Reply:
@micalevisk Great. Done.